runtime/util/fltconv.c

```diff
@@ -386,8 +387,72 @@ helperCConvertLongToFloat(I_64 src)
 {
 	jfloat tmpDst;
 
+/* Every platform but 32-bit Windows can use the cast.
+ * Using VS2017 15.9 the cast doesn't give the expected result.
```

Comment:
Shouldn't this say 19.16 (I don't know where 15.9 came from)?

Reply:
15.9 is the version of VS2017 we install to get the C compiler 19.16
https://docs.microsoft.com/en-us/visualstudio/releasenotes/vs2017-relnotes

Reply:
Ok, got it. Thanks.